packages/flutter/lib/src/rendering/layer.dart

```diff
@@ -79,23 +100,30 @@ abstract class Layer extends Object with TreeDiagnosticsMixin {
   @override
   void debugFillDescription(List<String> description) {
     super.debugFillDescription(description);
+    description.add('${ owner != null ? "owner: $owner" : "DETACHED" }');
     if (debugCreator != null)
       description.add('creator: $debugCreator');
   }
 }
 
-/// A composited layer containing a [Picture]
+/// A composited layer containing a [Picture].
+///
+/// Picture layers are always leaves in the layer tree.
 class PictureLayer extends Layer {
-  /// The picture recorded for this layer
+  /// The picture recorded for this layer.
+  ///
+  /// The picture's coodinate system matches this layer's coodinate system.
   ///
-  /// The picture's coodinate system matches this layer's coodinate system
+  /// The scene must be explicitly recomposited after this property is changed.
```

Comment:
I assume that the references to recompositing "the scene" here and below refer to text that begins "To composite the tree" in the Layer class doc.  It might be helpful to provide a more explicit link back to that.